### ISSUE TITLE
good way to clean up after an uninstall?

### ISSUE THREAD

**Issue by twilwa** [pkgxdev/pkgx#901]
I notice that pkgx mentions that it may have polluted .local during install. Is there a good way to find where the binaries out to be after removing pkgx? Since they're shims/tarballs, am I safe to just delete the whole thing, and does that mean I have to reinstall anything that pkgx was originally handling? struggling to get this uninstall completed, heh. 

If there is a good methodology, I might suggest writing/be able to write a PR myself detailing such a method to the documentation.

Thanks for the assist.

**Comment by jhheider** [comment]
pkgx uses .local/bin for `install`ed shims; on linux it uses .local/state and .cache (i believe) unless their respective XDG_* variables are set.

**Comment by twilwa** [comment]
yeah, I ended up with a grep rm and seeing what's broken, more or less, lol. There's probably a better way to do it, i'm still pretty green. anything i should be on the lookout for in terms of .local/state and .cache cleanup?

can close this out whenever unless there's a more complete series of uninstall commands you wanna spitball :)

**Comment by mxcl** [comment]
It's not that pkgx pollutes `~/.local` it's that the tools you run with `pkgx` may pollute your home directory in general.

You will need to refer to each and every tool to figure out what to do about it.

This is not a `pkgx` problem, all package managers have this problem.
